.github/workflows/system-tests.yml

```diff
@@ -48,6 +48,51 @@ jobs:
       - name: Run
         run: ./run.sh TRACER_ESSENTIAL_SCENARIOS
 
+      - name: Compress artifact
+        if: ${{ always() }}
+        run: tar -czvf artifact.tar.gz $(ls | grep logs)
+
+      - name: Upload artifact
+        uses: actions/upload-artifact@v2
+        if: ${{ always() }}
+        with:
+          name: logs_express-poc
+          path: artifact.tar.gz
+
+  system-tests-propagation:
```

Comment:
what is this `CROSSED_TRACING_LIBRARIES` scenario ? :thinking:

Reply:
🤷 I'm not sure, but it's there and was increasing the time here.